yarl/_url.py

```diff
@@ -435,6 +450,24 @@ def raw_authority(self):
         """
         return self._val.netloc
 
+    def _get_default_port(self) -> Optional[int]:
```

Comment:
Optional is discouraged because it's considered confusing:
```suggestion
    def _get_default_port(self) -> Union[int, None]:
```